libs/lb/lb-rs/src/model/errors.rs

```diff
@@ -86,6 +86,9 @@ impl Display for CoreError {
             CoreError::InvalidAuthDetails => {
                 write!(f, "our server failed to authenticate your request, please try again")
             }
+            CoreError::KeyPhrasesMistyped => {
```

Comment:
gonna revamp errors myself, so maybe we don't need to bother thinking too deep about this but Mistyped is like another word for invalid, we have a few variations of this sort of word already, idk feels weird

Reply:
I switched the name.